tests/test_working_embedding_parent_race.py

```
@@ -0,0 +1,94 @@
+"""Regression coverage for the trim-before-embedding parent race."""
+
+import json
+import logging
+import sqlite3
+
+import mnemosyne.core.beam as beam
```

Ruff (0.15.21)

[warning] Use `from mnemosyne.core import beam` in lieu of alias

Replace with `from mnemosyne.core import beam`

(PLR0402)